fusor/fusor.py

```diff
@@ -98,6 +108,36 @@ def add_sequence_id(self, fusion: Fusion,
                             component_genomic.location.sequence_id = \
                                 self.translate_identifier(location.sequence_id, target_namespace)  # noqa: E501
 
+    def add_gene_descriptor(self, fusion: Fusion) -> None:
+        """Add additional fields to `gene_descriptor` in fusion object
+
+        :param Fusion fusion: A valid Fusion object
+        """
+        for field in [fusion.protein_domains, fusion.structural_components,
+                      fusion.regulatory_elements]:
+            for obj in field:
+                if "gene_descriptor" in obj.__fields__.keys():
+                    norm_gene_descr = \
+                        self._normalized_gene_descriptor(obj.gene_descriptor.label)  # noqa: E501
+                    if norm_gene_descr:
+                        obj.gene_descriptor = norm_gene_descr
+
+    def _normalized_gene_descriptor(self,
+                                    query: str) -> Optional[GeneDescriptor]:
+        """Return gene descriptor from normalized response.
+
+        :param str query: Gene query
+        :return: Gene Descriptor for query
+        """
+        gene_norm_resp = self.gene_normalizer.normalize(query)
+        try:
+            match_type = gene_norm_resp["match_type"]
+        except TypeError:
+            match_type = gene_norm_resp.match_type
```

Comment:
I'm guessing you're about to do this, but we should probably just return a consistent type in the normalizer instead of doing this